Formula/profanity.rb

```diff
@@ -29,13 +31,18 @@ class Profanity < Formula
   depends_on "libotr" => :recommended
   depends_on "gpgme" => :recommended
   depends_on "terminal-notifier" => :optional
+  depends_on :python if build.with? "python-plugin-support"
```

Comment:
It works both, against system and brewed python.
This way it will compile against brewed version if installed, against system version otherwise.
I'm not shure what should be the recommended behaviour …